packages/node_modules/express-pouchdb/lib/routes/root.js

```diff
@@ -13,7 +13,8 @@ module.exports = function (app) {
   app.get('/', function (req, res) {
     var json = {
       'express-pouchdb': 'Welcome!',
-      'version': pkg.version
+      'version': pkg.version,
+      'pouchdb-adapters': req.PouchDB.preferredAdapters
```

Comment:
is `PouchDB.preferredAdapters` considered public API? Or could it happen that it might get removed in a non-breaking release? Just want to make sure, it’s currently not documented on https://pouchdb.com/api.html